docs/import/add_disaster.html

```diff
@@ -22,12 +22,15 @@
 
 <form action="" id="disaster-picker">
   <label for="disaster">Disaster: </label>
-  <select id="disaster" style="width: 160px"></select>
+  <select id="disaster" style="width: 160px">
```

Comment:
Why not just hide the whole select element? I think you can put it inside a <span> that keeps its width to avoid any jumping around. Are the dots actually pulling their weight?

Reply:
I've (not very tidily) changed it so it hides the disaster picker. Personally I like how the other way looks better, I think it flows more smoothly when the whole page doesn't change. But lmk if you have strong feelings otherwise!

Reply:
Why not implement my suggestion of wrapping the select in an element that keeps its width, so nothing else moves? Not on a computer right now, but sounds like that's not what you did?